proposals/0273-webengine-projection-mode.md

```diff
@@ -26,34 +26,25 @@ A new template called `WEB_VIEW` should be added. This template should have the
-4. The `WEB_VIEW` template is not allowed to to support textfields, softbuttons, graphics (inside or outside of the webview). DuplicateUpdatesFromWindowID=0 is not allowed for widgets.
+4. The `WEB_VIEW` template must not declare support in its WindowCapabilities for any softButtonCapabilities, the TextFields with name: mainField1-4, mediaClock, mediaTrack, nor the ImageFields with name: softButtonImage, menuIcon, graphic, or secondaryGraphic.
 
 If this template is used by WebEngine apps, the HMI of the IVI should present the WebEngine app and show the application's web page. 
 The application can control the web `document` object using JavaScript code to manipulate the document object model of the WebEngine app.
 JavaScript is the only language supported within current proposal.
 The HMI should respect the WebEngine app as the first responder to touch events. This means that touchable elements in the `document` should be accessible through the system's touch screen to the user.
-If the app sends RPCs which trigger a pop-up on the HMI screen, so this pop-up overlays what WEB_VIEW area shows (Alert, PerformInteraction etc.), touch events will first go through the pop-up area.
+If the app sends RPCs which trigger a pop-up on the HMI screen, so this pop-up overlays what WEB_VIEW area shows (`Alert`, `PerformInteraction` etc.), touch events will first go through the pop-up area.
 After pop-up is closed, touch events processing returns back to the WEB_VIEW area.
 
-The HMI should include a button to access the app list, the add command menu button (or a requirement for the app to implement ShowAppMenu or otherwise implement CloseApplication), and app/template title.
-The template/app title should be visible when the app is activated. The exact location of the template/app title relies on the OEM.
+1. The HMI should include an app title and a template title. The template/app title should be visible when the app is activated. The exact location of the template/app title relies on the OEM.
+2. The HMI should include a button to access the app list
+3. The HMI should not show a menu button on the HMI. Instead the app should be required to behave as followed:
 
+    3.1 . If the app uses AddCommand or AddSubMenu the app is required to provide an own menu button on the WEB_VIEW and send ShowAppMenu if this menu button is selected by the user.
 
-#### 1.1. Application deactivation
-The HMI should always support showing a menu button, but it would only be shown if the app sends at least one AddCommand or AddSubMenu. If the app does not send any AddCommands or AddSubmenus, then the menu button should not be shown, but in that case the app must implement the CloseApplication RPC in an **EXIT** button somewhere in their in-app UI.
+    3.2. If the app does not use commands or sub menus the app is required to provide a way to the user to exit the application. The app should use CloseApplication RPC for this.

```

Comment:
Please add backticks as shown in https://github.com/smartdevicelink/sdl_evolution/issues/912#issuecomment-637749152.

Reply:
@theresalech Thank you. Done in 96ace2f161629c16b1280f18c852cccb6557758c